### ISSUE TITLE
Refuelling troubles

### ISSUE THREAD

**Issue by l0k1** [l0k1/A-10#2]
After disconnect, plane cant reconnect for AAR.

**Comment by onox** [comment]
Is this still an issue?

**Comment by l0k1** [comment]
Yes - a lot of the systems (including FDM) are currently under a rewrite, so this'll get looked at more in depth in the near future.
